kits/paper-to-project/agent.md

```
@@ -0,0 +1,21 @@
+4. Report Generator
+
+## Input
+
+- Research paper abstract, summary, or full text
+
+## Output
+
+- Research analysis
+- Implementation roadmap
+- Development plan
+- Professional markdown report
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)